File: API/ios-tms.Core/Domain/ShipmentItem.cs
using System;
namespace iOS_TMS.Core.Domain;

public class ShipmentItem : EntityBase
{
    public int ShipmentId { get; set; }
    public int Units { get; set; }
    public decimal Weight { get; set; }
    public string Description { get; set; }
    public virtual Shipment Shipment { get; set; }
}

File: App/ios-tms/Models/ShipmentItem.swift
//
//  ShipmentItem.swift
//  ios-tms
//
//  Created by Kyle Morton on 8/28/22.
//

import Foundation

struct ShipmentItem : Codable, Identifiable {
    var id: Int
    var shipmentId: Int
    var units: Int
    var weight: Decimal
    var description: String
    
    init(id: Int, units: Int, weight: Decimal) {
        self.id = id
        self.units = units
        self.weight = weight
    }
}
